test/tournament_test.exs

```diff
@@ -213,9 +202,12 @@ defmodule TournamentTest do
     # Invalid input lines in an otherwise-valid game still results in valid
     # output.
     input = [
+      # FIXME: Investigate filtering out empty lines, be sure to trim them!
```

Comment:
Some ideas if I had more time to fix this test!
I'd also maybe break this up into several tests.